binding/vue/src/rhino.ts

```diff
@@ -51,117 +50,94 @@ export default {
-         * @param rhinoFactory The language-specific worker factory
-         * @param inferenceCallback A method invoked upon completion of intent inference.
-         * @param contextCallback A method invoked after context information is ready.
-         * @param readyCallback A method invoked after Rhino has initialized.
-         * @param errorCallback A method invoked if an error occurs within `PorcupineWorkerFactory`.
-         */
-        async init(
-          rhinoFactoryArgs,
-          rhinoFactory,
-          inferenceCallback = (_: RhinoInference) => {},
-          contextCallback = (_: string) => {},
-          readyCallback = () => {},
-          errorCallback = (error: Error) => {console.error(error)}
-        ) {
-          try {
-            const { accessKey, context, endpointDurationSec, requireEndpoint, start } = rhinoFactoryArgs;
-            this.$_rhnWorker_ = await rhinoFactory.create({
-              accessKey,
-              context: JSON.parse(JSON.stringify(context)),
-              endpointDurationSec,
-              requireEndpoint,
-              start
-            });
-            this.$_webVp_ = await WebVoiceProcessor.init({
-              engines: [this.$_rhnWorker_],
-            });
-    
-            this.$_rhnWorker_.onmessage = messageEvent => {
-              switch (messageEvent.data.command) {
-                case 'rhn-inference':
-                  inferenceCallback(messageEvent.data.inference);
-                  // Reset Push-to-Talk
-                  this.$_rhnWorker_?.postMessage({ command: 'pause' });
-                  break;
-                case 'rhn-info':
-                  const info = messageEvent.data.info;
-                  contextCallback(info);
-                  break;
-              }
-            };
-            this.$_rhnWorker_.postMessage({ command: 'info' });
-            readyCallback();
-          } catch (error) {
-            errorCallback(error as Error);
-          }
+        $_rhino_: null as RhinoWorker | null,
+        isLoadedCallback: function (): void {
+          return;
         },
-        /**
-         * Start processing audio.
-         */
-        async start() {
-          if (this.$_webVp_ !== null) {
-            await this.$_webVp_.start();
-            return true;
-          } else {
-            return false;
-          }
+        isListeningCallback: function (): void {
+          return;
         },
-         /**
-         * Start processing audio.
-         */
-        async stop() {
-          if (this.$_webVp_ !== null) {
-            await this.$_webVp_.stop();
-            return true;
-          } else {
-            return false;
+        errorCallback: function (): void {
+          return;
+        },
+        async init(
+          accessKey: string,
+          context: RhinoContext,
+          inferenceCallback: InferenceCallback,
+          model: RhinoModel,
+          contextCallback: (info: string) => void,
```

Comment:
I think this should be called contextInfoCallback. All of our other SDKs call the property contextInfo, so this callback should also